Replit Guide

kyle-wordpress opened the pull request:
I made a Replit guide for ModMail. No images but very descriptive and easy to follow.

kyle-wordpress committed:
Create replit guide.md

kyle-wordpress committed:
Created the steps.

kyle-wordpress committed:
Update and rename replit guide.md to Replit Guide.md

kyle-wordpress committed:
Update Replit Guide.md

kyle-wordpress committed:
Update Replit Guide.md

kyle-wordpress committed:
Update Replit Guide.md

zayKenyon commented:
Isn't this a bad idea due to Repl.it resetting their databases whenever they restart a container?

kyle-wordpress commented:
> Isn't this a bad idea due to Repl.it resetting their databases whenever they restart a container?

Read the document. I specifically put in why this occurs and how to fix this.

>Why are you using MySQL, not the inbuilt SQLite?
> - Due to how Replit operates, it resets the database to a previous version every few hours or so. The SQLite database is located directly on the Repl, but the MySQL database is external and therefore can't be rolledback. You could skip Steps 7-9 completely, but there are downsides if you have ModMail threads that last a long time, as the channels may be disconnected and you may lose logs.

kyle-wordpress committed:
Update Replit Guide.md

Added support for v3.6.1, re-ordered the guide to make it more user friendly and other tweaks.

kyle-wordpress committed:
Update Replit Guide.md

Added a couple images to confusing parts.

kyle-wordpress commented:
@Dragory I'm done for now with the guide. I'd appreciate if you would review it and add it to help others!

kyle-wordpress committed:
Update Replit Guide.md

Minor spelling mistake

kyle-wordpress committed:
Update README.md

added replit guide to the list
